proto/signer.proto

```diff
@@ -22,9 +22,6 @@ service KeyManagement {
 service Signer {
   // Sign calculates a cryptographic signature using the Key associated with a KeyID and returns the signature
   rpc Sign(SignatureRequest) returns (Signature) {}
-
-  // CheckHealth returns the HealthStatus with the service
-  rpc CheckHealth(Void) returns (HealthStatus) {}
```

Comment:
It just seemed redundant because they were the same server, and seem like they always will be.  I don't mind having it in both, or splitting it off entirely into its own service (I think grpc now has its own health service too, but we might have to upgrade).